.github/workflows/ci.yml

```diff
@@ -1,5 +1,5 @@
 name: CI
-on: [push]
+on: [push, pull_request]
```

Comment:
I've updated `master` branch protection rules to require checks before merging, which has the same effect. Good to have it configured here too I guess :)

Reply:
Oh cool, does that make the checks run on forks?